### ISSUE TITLE
Can't reexport the named export 'attributesToProps' from non EcmaScript module (only default export is available)

### ISSUE THREAD

**Issue by anhtran** [remarkablemark/html-react-parser#217]
I used this in Next.js. After upgrade to the new version 1.2.2, the error has occurred.

```
./node_modules/html-react-parser/index.mjs
Can't reexport the named export 'attributesToProps' from non EcmaScript module (only default export is available)
```

**Comment by varunkumar3091999** [comment]
Same issue for me too.

**Comment by remarkablemark** [comment]
I can reproduce the error in #219, I will work on a fix. This is caused by #216.

**Comment by remarkablemark** [comment]
Published [v1.2.3](https://github.com/remarkablemark/html-react-parser/releases/tag/v1.2.3):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm i html-react-parser@1.2.3
```

[Yarn](https://yarnpkg.com/package/html-react-parser):

```sh
yarn add html-react-parser@1.2.3
```
